include/aspect/utilities.h

```diff
@@ -141,6 +141,15 @@ namespace aspect
                            const dealii::Point<2> &point);
 
     /**
+     * Given a 2d point and a list of points which form a polygon, compute the smallest
+     * distance of the point to the polygon.
```

Comment:
can you add how the sign is defined, namely positive if the point is inside, and negative if outside?